webpack/assets/javascripts/react_app/components/Pagination/Pagination.js

```diff
@@ -1,97 +1,9 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { isEmpty } from 'lodash';
-import { Paginator } from 'patternfly-react';
-import { translate as __ } from '../../common/I18n';
-import { usePaginationOptions } from './PaginationHooks';
-import {
-  getURIpage,
-  getURIperPage,
-  changeQuery,
-} from '../../common/urlHelpers';
-import { useForemanSettings } from '../../Root/Context/ForemanContext';
-import './pagination.scss';
+import { deprecate } from '../../common/DeprecationService';
 
-const Pagination = props => {
-  const {
-    data,
-    pagination,
-    onPageSet,
-    onPerPageSelect,
-    dropdownButtonId,
-    disableNext,
-    disablePrev,
-    ...otherProps
-  } = props;
-
-  const { perPage } = useForemanSettings();
-  const perPageOptions = usePaginationOptions();
-  const urlPage = getURIpage();
-  const urlPerPage = getURIperPage() || null;
-  const className = isEmpty(data.classNames)
-    ? 'col-md-12'
-    : `col-md-12 ${data.classNames.pagination_classes}`;
-
-  const pageOpts = {
-    page: urlPage,
-    perPage: urlPerPage || perPage,
-    perPageOptions,
-    ...pagination,
-  };
-
-  const messages = {
-    firstPage: __('First Page'),
-    previousPage: __('Previous Page'),
-    currentPage: __('Current Page'),
-    nextPage: __('Next Page'),
-    lastPage: __('Last Page'),
-    perPage: __('per page'),
-    of: __('of'),
-  };
-
-  return (
-    <Paginator
-      pagination={pageOpts}
-      viewType={data.viewType}
-      itemCount={data.itemCount}
-      onPageSet={onPageSet}
-      onPerPageSelect={onPerPageSelect}
-      disableNext={disableNext}
-      disablePrev={disablePrev}
-      className={className}
-      dropdownButtonId={dropdownButtonId}
-      messages={messages}
-      {...otherProps}
-    />
+export default () => {
+  deprecate(
+    'PF3 Pagination',
+    3.2,
+    'PF4 pagination from "react_app/components/Pagination/index.js" instead'
```

Comment:
@ezr-ondrej I added 3 deprecation warnings in the files I removed just so the bundle will still build successfully for plugins.

Reply:
`"instead"` is already added in the deprecation service.